www/js/survey/enketo/enketo-add-note-button.js

```diff
@@ -66,16 +65,14 @@ angular.module('emission.survey.enketo.add-note-button',
     // Enketo requires these specific date/time formats
     if (momentBegin) {
       prefills.Start_date = momentBegin.format('YYYY-MM-DD');
-      prefills.Start_time = momentBegin.format('HH:mm:ss.SSSZ');
+      prefills.Start_time = momentBegin.format('HH:mm:ss.SSS');
```

Comment:
what is this change for? It was introduced in https://github.com/JGreenlee/e-mission-phone/commit/abdb5c640b71229026d4af32f77c95b65e5f7455 but doesn't seem to be related..

Reply:
Ah, that change should have been in ab6f7adee12ee929ef380da554df5b1d523f3daa.
Regardless, I changed it again last night to actually fix the prefilled timezones.